tests/deploy/cloudbeat-pytest.yml

```diff
@@ -296,6 +296,9 @@ spec:
           requests:
             cpu: 100m
             memory: 100Mi
+          limits:
+            cpu: 1000m
+            memory: 1Gi
```

Comment:
@DaveSys911 nit, we can fail to catch real performance problem defining memory limit to 1Gi.
I think we need to measure cpu and memory and define more real limit value for Cloudbeat.

Reply:
I agree, this one is a very generous upper limit to avoid killing CI workers. We should decrease it.